charts/et-sya/values.yaml

```diff
@@ -21,6 +21,6 @@ nodejs:
     PCQ_URL: 'https://pcq.{{ .Values.global.environment }}.platform.hmcts.net/service-endpoint'
     PCQ_HEALTH_URL: 'https://pcq.{{ .Values.global.environment }}.platform.hmcts.net/health'
     PCQ_ENABLED: 'true'
-    ET1_BASE_URL: 'https://et-stg-azure.staging.et.dsd.io'
+    ET1_BASE_URL: 'https://et-pet-et1.{{ .Values.global.environment }}.platform.hmcts.net'
```

Comment:
**Note** - This is overridden in Flux for Prod - https://github.com/hmcts/cnp-flux-config/blob/1c0c2dc5ccdf53c25ed729918dd545a932fbbfb3/apps/et/et-sya/prod.yaml#L18